client/my-sites/earn/home.tsx

```diff
@@ -453,11 +459,16 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 					"Check out your ad earnings history, including total earnings, total paid to date, and the amount that you've still yet to be paid."
 			  )
 			: translate(
-					'Make money each time someone visits your site by displaying advertisements on all your posts and pages. {{em}}Available only with a Premium, Business, or eCommerce plan{{/em}}.',
+					'Make money each time someone visits your site by displaying advertisements on all your posts and pages. {{em}}Available only with a %(planNames)s plan{{/em}}.',
```

Comment:
Eventhough we use these new `Premium or Professional`-like translations only in this context, we should try to avoid building sentences by injecting translations. In this we can do something like this:
```suggestion
					'Make money each time someone visits your site by displaying advertisements on all your posts and pages. {{em}}Available only in these plans: %(planNames)s{{/em}}.',
```

Reply:
Actually, we can split off the `Available only in these plans:` construct into a separate translation to be reused.

Reply:
Not sure I follow here. Are you suggesting removing the italics, or creating something like this?

```jsx
	const getPlanNames = () => {
		return isNonAtomicJetpack
			? translate( 'Available only with a Premium or Professional plan' ) : translate( 'Available only with a Premium, Business, or eCommerce plan' );
	}
```

Reply:
> creating something like this?

Yup, something like that, so the translation no longer needs to include the italics.

Example:


```js
const getAvailablePlans = () =>
  isNonAtomicJetpack
    ? translate( 'Available only with a Premium or Professional plan' )
    : translate( 'Available only with a Premium, Business, or eCommerce plan' );

const body = (
  <>
    translate( 'Accept credit card payments via PayPal for physical products, digital goods, services, donations, or support of your creative work.' )
    { ! hasSimplePayments && <em>{ getAvailablePlans() }</em> }
  </>
);
```

Reply:
Thanks, I think that I've understood it! Let me know if the approach I've committed works. :)